src/main/java/com/synopsys/integration/alert/provider/blackduck/BlackDuckValidator.java

```diff
@@ -43,15 +41,13 @@
 import com.synopsys.integration.log.IntLogger;
 import com.synopsys.integration.log.Slf4jIntLogger;
 
-@Component
 public class BlackDuckValidator extends ProviderValidator {
```

Comment:
Should we add a TODO here?

Reply:
I have created a Factory class to instantiate the validator.

Reply:
I've removed the need for the factory